controllers/index.js

```diff
@@ -0,0 +1,13 @@
+const endpoints = require("../endpoints.json");
+const fs = require("fs");
+
+exports.getEndpoints = (req, res, next) => {
+  fs.readFile(
```

Comment:
Nice approach using fs. As you've already imported endpoints one line 1 though, how about just returning it directly in your response?